### DISCUSSION TITLE
HTTPSConnectionPool(host='paddleocr.bj.bcebos.com', port=443)

### DISCUSSION THREAD

**Discussion by DanBrown47** [PaddlePaddle/PaddleOCR#13043] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu 22.04 乌班图 22.04
- 版本号/Version：Paddle： paddlepaddle-gpu==2.5.2.post102  
- PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：paddleocr --image_dir ./Document.png --det flase --lang en --use_gpu false
- 完整报错/Complete Error Message：requests.exceptions.ConnectionError: HTTPSConnectionPool(host='paddleocr.bj.bcebos.com', port=443): Max retries exceeded with url: /PP-OCRv3/english/en_PP-OCRv3_det_infer.tar (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb475290be0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by tink2123** [comment]
Does your machine support networking? Please try turning off all proxies
